internal/kafka/internal/services/data_plane_cluster_test.go

```diff
@@ -77,6 +77,79 @@ func Test_DataPlaneCluster_UpdateDataPlaneClusterStatus(t *testing.T) {
 				return NewDataPlaneClusterService(sampleValidApplicationConfigForDataPlaneClusterTest(clusterService))
 			},
 		},
+		{
+			name:          "An error is returned when svcErr != nil",
```

Comment:
This description is too implementation specific. A better name would be one that describes the test case instead:
An error is returned when an error happens when trying to get the cluster from the provided cluster id